kratos/tests/geometries/test_quadrilateral_3d_4.cpp

```diff
@@ -19,6 +19,7 @@
 #include "testing/testing.h"
 #include "geometries/quadrilateral_3d_4.h"
 #include "tests/geometries/test_geometry.h"
+#include "includes/global_variables.h"
```

Comment:
Why has this been included?

Reply:
I don't know about this change. I can not remember if I made it, but I guess I don't. It is possible that this line was there before and it was cleaned but in some merge remained there in my local branch. However it is strange. If it is not needed it must be taken away.

Reply:
@josep-m-carbonell are u still using GitKraken? i am asking since we are reaching the conclusion that it is too easy to make mistakes when using it, since it brings to one click "strange" operations.